### PR TITLE
[H3] Add new port

### PR DESCRIPTION
**Pull request by kreuzerkrieg**
A Hexagonal Hierarchical Geospatial Indexing System

- Which triplets are supported/not supported?
Tested with linux, theoretically should work on Win/OSX, no arm support I guess
- Have you updated the CI baseline?
Yes

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

File: ports/h3/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+vcpkg_install_cmake()
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
+
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin ${CURRENT_PACKAGES_DIR}/debug/include)
+
+file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT})
+file(RENAME ${CURRENT_PACKAGES_DIR}/share/${PORT}/LICENSE ${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright)
```
Comment (PhoebeHui) [inline ports/h3/portfile.cmake]: Could you update it to 
file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@kreuzerkrieg, thanks for the PR!

From portfile.cmake file, It looks H3 doesn't support arm, arm64, and uwp, could you add them to https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt

**Comment by kreuzerkrieg** [comment]
> @kreuzerkrieg, thanks for the PR!
> 
> From portfile.cmake file, It looks H3 doesn't support arm, arm64, and uwp, could you add them to https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt

should it be `ignore`?

**Comment by PhoebeHui** [comment]
I think 'fail' would make more sence, 'Ignore' is fine,

**Comment by kreuzerkrieg** [comment]
> I think 'fail' would make more sence, 'Ignore' is fine,

I dont mind to change it.
Pushed

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

**Comment by isaacbrodsky** [comment]
I'm not aware of an issue with H3 on arm64 (I had previously tested it on arm but we don't have ci for that). Is it just untested on arm or was there an issue with it?

**Comment by kreuzerkrieg** [comment]
> I'm not aware of an issue with H3 on arm64 (I had previously tested it on arm but we don't have ci for that). Is it just untested on arm or was there an issue with it?

Yes, I dont have any hardware to check it with arm, so, to be on the safe side I just disabled it. I can enable it, but if something will go wrong I cant test it.

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Comment by kreuzerkrieg** [comment]
> I'm not aware of an issue with H3 on arm64 (I had previously tested it on arm but we don't have ci for that). Is it just untested on arm or was there an issue with it?

BTW, Isaac, if you have time and hardware you are more than welcome to apply your changes regarding the `arm` stuff

**Pull request merged**